src/simulation_strategies/FixedLossSimulationStrategy.py

```diff
@@ -46,35 +47,24 @@ def __randomize_modes_phases(state_in_modes_basis: ndarray) -> ndarray:
             :param state_in_modes_basis: A given state in modes basis.
             :return: Given mode state with randomized phases.
         """
-        randomized_phases_state = []
-
-        for mode in state_in_modes_basis:
-            phi = random() * 2 * pi
-            randomized_phases_state.append(exp(1j * phi) * mode)
-
+        randomized_phases_state = exp(1j * rand(len(state_in_modes_basis))) * state_in_modes_basis
```

Comment:
Expression could be rewritten like this for convenience.

Reply:
This is actually correct and definitely looks more pythonic. Thank you. +1